lib/cfn-model/parser/ec2_network_acl_parser.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative 'parser_error'
+require 'cfn-model/model/ec2_network_acl'
+require 'cfn-model/model/references'
+
+class Ec2NetworkAclParser
+  def parse(cfn_model:, resource:)
+    network_acl = resource
+
+    attach_nacl_entries_to_nacl(cfn_model: cfn_model, network_acl: network_acl)
+    network_acl
+  end
+
+  private
+
+  def egress_network_acl_entries(cfn_model)
+    network_acl_entries = cfn_model.resources_by_type 'AWS::EC2::NetworkAclEntry'
+    network_acl_entries.select(&:egress)
+  end
+
+  def ingress_network_acl_entries(cfn_model)
+    network_acl_entries = cfn_model.resources_by_type 'AWS::EC2::NetworkAclEntry'
+    network_acl_entries.select do |network_acl_entry|
+      network_acl_entry.egress.nil? || !network_acl_entry.egress
```

Comment:
beware of "weak typing" in cfn.  this could be "true", true, "false", false.  there should be a truthy? util in the model you can use instead of a straight boolean op

Reply:
There isn't currently a truthy? util in cfn_model, just cfn_nag, but I can add one to model as part of this PR.